DXMainClient/DXGUI/Multiplayer/GameLobby/MultiplayerGameLobby.cs

```diff
@@ -521,7 +522,14 @@ private void RollDiceCommand(string dieType)
 
             BroadcastDiceRoll(dieSides, results);
         }
-
+        private void HandRPSCommand(string ent)
```

Comment:
Should choose a parameter name that is easy to understand. `ent` confuses people.

Reply:
ent is a string that represents pressing Enter similar to a blank input